java/kotlin-extractor/src/main/kotlin/utils/versions/v_1_7_20-Beta/allOverridden.kt

```diff
@@ -0,0 +1,6 @@
+package com.github.codeql.utils.versions
+
+import org.jetbrains.kotlin.ir.declarations.IrSimpleFunction
+import org.jetbrains.kotlin.ir.util.allOverridden
+
+fun IrSimpleFunction.allOverridden(includeSelf: Boolean = false) = this.allOverridden(includeSelf)
```

Comment:
Does anything stop this recursively calling itself? I think it would be clearer if we changed the name of our function.

Reply:
It doesn't call itself recursively, but I only know this from the IDE, and don't know the exact symbol resolution rules.

I changed the name to be more explicit.